src/modules/logicModule/LM_PC_KPIRewarder_v1.sol

```diff
@@ -364,6 +371,9 @@ contract LM_PC_KPIRewarder_v1 is
             assertionData[assertionId].asserter,
             assertionId
         );
+
+        // Independently of the fact that the assertion resolved true or not, new assertions can now be posted.
+        assertionPending = false;
```

Comment:
Just out of interest:
is there a usecase where a assertion cant be resolved?

Reply:
Otherwise LGTM

Reply:
No, it ends up resolved truthfully or not. It can be disputed though, which delays the process